main.js

```diff
@@ -12,11 +12,40 @@ import {parseRequest, parseSignatureHeader} from 'http-signature-header';
 import {CapabilityInvocation} from 'ocapld';
 import {TextDecoder, TextEncoder, base64Decode} from './util.js';
 
+/**
+ * Verifies a zcap in the form of an http-signature header.
+ *
+ * @param {object} options - Options to use.
+ * @param {URL} options.url - The url of the request.
```

Comment:
Isn't this a string?

```suggestion
 * @param {string} options.url - The url of the request.
```